docker/cleanspeak/cleanspeak/Dockerfile

```diff
@@ -13,27 +13,27 @@
 #  > docker run -p 8001:8001 -it cleanspeak/cleanspeak-webservice
```

Comment:
update this to `docker run -p 8001:8001 -it cleanspeak/cleanspeak-app`?

Reply:
Sorry about the sloppiness here (lots of branches/files/PRS). Fixed.